.github/workflows/new-cfe-notifications.yml

```diff
@@ -3,12 +3,41 @@ on:
   issues:
     types: [labeled]
 
-permissions:
-  issues: write
-  contents: write
-  pull-requests: write
+permissions: {}
 
 jobs:
+  prep:
+    if: github.event.label.name == 'CFE Approved' || github.event.label.name == 'CFE Rejected'
+    runs-on: ubuntu-20.04
+    outputs:
+      release_number: ${{ steps.extract-release.outputs.RELEASE_NUMBER }}
+    steps:
+      - name: Extract Release Number from Issue Body
+        id: extract-release
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const event = ${{ toJSON( github.event ) }}
```

Comment:
Is this actually needed? Or can we use directly `github.event.issue.body` on the statement below?

Reply:
Good call, used context object instead: https://github.com/woocommerce/woocommerce/pull/52002/commits/94e350163d5d43fcb34fadb4cebd4c27f8eb6568